frontend/libs/studio-components/src/components/StudioCodelistEditor/StudioCodeListEditorRow/StudioCodeListEditorRow.tsx

```
@@ -105,45 +105,141 @@ export function StudioCodeListEditorRow({
+        onFocus={handleFocus}
+        ref={ref}
+      />
+    );
+  }
 }
 
+const NumberfieldCell = forwardRef<HTMLInputElement, InputCellProps>(
+  ({ label, value, onChange, onFocus, autoComplete }, ref) => {
+    const handleNumberChange = useCallback(
+      (numberValue: number): void => {
+        if (isNaN(numberValue)) return;
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)